kbase-extension/static/kbase/css/appCell.css

```diff
@@ -4,11 +4,11 @@
 */
 
 .kb-app-cell {
-  font-family: 'Oxygen', sans-serif;
+  font-family: Oxygen, Arial, sans-serif;
```

Comment:
standardised the font declarations and added in fallback fonts where they were absent